src/machine/asm/execute_x64.S

```diff
@@ -467,49 +468,14 @@ ckb_vm_x64_execute:
 .CKB_VM_ASM_LABEL_OP_CUSTOM_ASM_TRACE_JUMP:
   /* Load current instruction as the full trace address */
   movq -16(INST_ARGS), TRACE
-  /* Prefetch trace info for the consecutive block */
-  prefetcht2 CKB_VM_ASM_TRACE_OFFSET_THREADS(TRACE)
-  mov CKB_VM_ASM_TRACE_OFFSET_LENGTH(TRACE), %edx
-  movq CKB_VM_ASM_ASM_CORE_MACHINE_OFFSET_CYCLES(MACHINE), %rax
-  addq CKB_VM_ASM_TRACE_OFFSET_CYCLES(TRACE), %rax
-  jc .exit_cycles_overflow
-  cmp CKB_VM_ASM_ASM_CORE_MACHINE_OFFSET_MAX_CYCLES(MACHINE), %rax
-  ja .exit_max_cycles_exceeded
-  movq %rax, CKB_VM_ASM_ASM_CORE_MACHINE_OFFSET_CYCLES(MACHINE)
-  addq %rdx, PC_ADDRESS
-  lea CKB_VM_ASM_TRACE_OFFSET_THREADS(TRACE), INST_PC
-  mov INST_PC, INST_ARGS
-  add $8, INST_ARGS
-  NEXT_INST
+  jmp .trace_loaded
```

Comment:
I found that the old code will never jump to the .exit_trace branch, but after the modification, it may jump to the .exit_trace branch through .trace_loaded. 

(I can't confirm whether this jump is possible now).

Reply:
So when `TRACE_JUMP` is generated, the length of trace will always be positive, in this sense it will never taken the jump at line 448.

Ideally for `TRACE_JUMP` we don't need tests on line 447-448 but they are just trivial to run, won't cause any noticeable difference.